packages/loader/src/MeshLoader.ts

```
@@ -29,3 +32,31 @@
     });
   }
 }
+
+class MeshContentRestorer extends ContentRestorer<ModelMesh> {
+  constructor(
+    resource: ModelMesh,
+    public url: string,
+    public requestConfig: RequestConfig,
```

ESLint

[error] Delete `,`

(prettier/prettier)